src/services/data-pool/data-pool-service.ts

```diff
@@ -10,11 +10,10 @@ class DataPoolService {
         const importReport = await dataPoolApi.executeDataPoolsBatchImport(requestFileContent);
         const importReportString = JSON.stringify(importReport, null, 4);
 
-        logger.info("Data Pools batch import succeeded!");
         if (outputToJsonFile) {
             const reportFileName = "batch_import_report_" + uuidv4() + ".json";
             fileService.writeToFileWithGivenName(importReportString, reportFileName);
-            logger.info("Batch import report file: " + reportFileName);
+            logger.info(FileService.fileDownloadedMessage + reportFileName);
```

Comment:
I honestly don't think the current message in this variable is too intuitive. 
`"File downloaded successfully. New filename: "`
The natural question is: Why was file downloaded if I just did an import? 
Also it is too generic.

I think having "Batch import report file" is more clear and tailored for this particular context.

Reply:
true I did revert it